lib/Access.php

```diff
@@ -1086,7 +1086,7 @@ private function processPagedSearchStatus($sr, $filter, $base, $iFoundItems, $li
 			foreach ($sr as $key => $res) {
 				if ($this->getLDAP()->controlPagedResultResponse($cr, $res, $cookie, $estimated)) {
 					\OC::$server->getLogger()->debug(
-						'Page response cookie '.$this->cookie2str($cookie).", estimated<$estimated>",
+						'Page response cookie '.$this->cookie2str($cookie)." at $limit/$offset, estimated<$estimated>",
```

Comment:
`at $offset/$limit` ? 500/2000 seems better.

Reply:
the order is always limit then offset. in all api calls, in SQL it is `SELECT ... LIMIT 10 OFFSET 1000` ...

Reply:
I'd bet there are a lot of people that just by reading "Page response cookie 0x00000 at 50/300" won't know what you're talking about. You HAVE TO know that the first number is the limit and the second is the offset, but if you don't know that most of the people will think that we're on point 50 of a maximum of 300.
I changed this to a range, like "Page response cookie 0x00000 at 300-350" which is more natural to read.